app/models/food.rb

```diff
@@ -0,0 +1,6 @@
+class Food < ApplicationRecord
+  has_many :recipe_foods
+  has_many :inventory_foods
+  belongs_to :user
+  has_many :recipe_foods
```

Comment:
You have done a great job implementing relationships.  I suggest checking ``has_many :recipe_foods`` because it looks duplicated